.github/workflows/notify-release.yaml

```diff
@@ -0,0 +1,70 @@
+name: Notify Release
+
+on:
+  workflow_call:
+    inputs:
+      release_version:
+        description: "The release version to notify about"
+        required: true
+        type: string
+      previous_tag:
+        description: "The previous tag for comparison"
+        required: true
+        type: string
+      ref:
+        description: "The git ref to checkout"
+        required: true
+        type: string
+
+jobs:
+  notify_release:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ inputs.ref }}
+      - id: get_base_branch
```

Comment:
This should probably be moved to reusable actions.

Reply:
Will do as a future improvement